beater/lsbeat.go

```diff
@@ -75,13 +75,13 @@ func (bt *Lsbeat) listDir(dirFile string, beatname string, init bool) {
 		//fmt.Println(f.Name(), dirFile+"/"+f.Name(), f.IsDir(), t, f.Size())
 
 		event := common.MapStr{
-			"@timestamp":  common.Time(time.Now()),
-			"type":        beatname,
-			"modTime":     common.Time(t),
-			"filename":    f.Name(),
-			"fullname":    dirFile + "/" + f.Name(),
-			"isDirectory": f.IsDir(),
-			"fileSize":    f.Size(),
+			"@timestamp":   common.Time(time.Now()),
+			"type":         beatname,
+			"mod_time":     common.Time(t),
+			"file_name":    f.Name(),
+			"full_name":    dirFile + "/" + f.Name(),
+			"is_directory": f.IsDir(),
```

Comment:
@monicasarbu No rule exists yet, but we could abbreviate `directory` always by `dir`: https://github.com/elastic/beats/blob/master/metricbeat/module/doc.go#L28 Or are there other abbreviations that could conflict with?